node-graph/graph-craft/src/proto.rs

```diff
@@ -687,17 +694,25 @@ mod test {
 	#[test]
 	fn topological_sort() {
 		let construction_network = test_network();
-		let sorted = construction_network.topological_sort();
+		let sorted = construction_network.topological_sort().unwrap();
```

Comment:
This should be an `.expect`

Reply:
that just makes it a bit easier to debug when tests fail

Reply:
Thanks, I've added descriptive error messages with `expect()` in my tests instead of using `unwrap()`